my-app/src/Pages/Home.tsx

```diff
@@ -9,11 +9,12 @@ const Home: React.FC<{}> = (props: any) => {
   const [page, setPage] = useState(1);
   const [search, setSearch] = useState("");
   const debouncedSearch = useDebounce(search, 750);
+  const [selectedGenre, setSelectedGenre] = useState("");
 
   const { isError, isLoading, data, isFetching, isPreviousData } = useQuery(
-    ["movie", debouncedSearch],
+    ["movie", debouncedSearch, selectedGenre],
```

Comment:
Let our dependency array be ['movies'] as we are fetching multiple movies and it will be easier for us to invalidate it